OSPSuite.Presentation.Importer/Core/DataFormat/ColumnsDataFormat.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace OSPSuite.Presentation.Importer.Core.DataFormat
+{
+   public class ColumnsDataFormat : IDataFormat
+   {
+      public string Name => "ColumnsFormat";
+
+      public string Description => "Simple format with data identified by column data...";
+
+      public IList<DataFormatParameter> Parameters { get; private set; }
+
+      public bool CheckFile(IUnformattedData data)
+      {
+         if (data.Headers.Where(h => h.Value.Level == ColumnDescription.MeasurementLevel.Numeric).Count() < 2)
+            return false;
+         setParameters(data);
+         return true;
+      }
+
+      private void setParameters(IUnformattedData data)
+      {
+         var keys = data.Headers.Keys.ToList();
+         Parameters = new List<DataFormatParameter>();
+
+         var missingKeys = new List<string>();
+
+         extractQualifiedHeadings(keys, missingKeys);
+         extractNonQualifiedHeadings(keys, missingKeys, data);
+         extractGeneralParameters(keys, data);
+      }
+
+      private string extractUnits(string description)
+      {
+         var units = Regex.Match(description, @"\[.+\]").Value;
```

Comment:
Please implement directly support for multiple units def such as [] or ()

Reply:
Again, this is a very basic class just to test the whole logic but I would not go into any more details until the real formats have been agreed upon